subsys/net/lib/aws_fota/src/aws_fota.c

```diff
@@ -361,10 +394,15 @@ int aws_fota_mqtt_evt_handler(struct mqtt_client *const client,
 
 		err = aws_jobs_subscribe_notify_next(client);
 		if (err) {
-			LOG_ERR("Unable to subscribe to notify next topic");
+			LOG_ERR("Unable to subscribe to notify-next topic");
 			return err;
 		}
 
+		err = aws_jobs_subscribe_job_id_get_topic(client, "$next");
+		if (err) {
+			LOG_ERR("Unable to subscribe to jobs/$next/get");
```

Comment:
Why don't we return here when we return in line 398 ?

Reply:
It was due to a previous idea where this was supposed to be optional so that you wouldn't disconnect due to not being able to subscribe to this topic. I don't think it is a good idea anymore so I'll add it back.